docs/kubernetes/ingress-usage.md

```diff
@@ -141,8 +141,41 @@ This example shows how to add predicates and filters:
 
 ## Custom Routes
 
-Custom routes is a way of extending the default routes configured for an
-ingress resource.
+Custom routes is a way of extending the default routes configured for
+an ingress resource.
+
+Sometimes you just want to return a header, redirect or even static
+html content. You can return from skipper without doing a proxy call
+to a backend, if you end your filter chain with `<shunt>`. The use of
+`<shunt>` requires, that you also use the `status()` filter.  To match
+your custom route with higher priority than your ingress you also have
+to add another predicate, for example the [Method("GET") predicate](/predicates/#method)
+to match the route with higher priority.
+
+Custom routes specified in ingress will always add the `Host()`
+[predicate](/predicates/#host) to match the host header specified in
+the ingress `rules:`. If there is a `path:` definition in your
+ingress, then it will based on the skipper CLI Parameter
```

Comment:
"then, based on the skipper CLI Parameter, it will set one of these predicates:"